cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,598 @@
+        view = navigate_to(self, "Add")
+        view.general.fill({"infra_map": infra_map, "name": name, "description": description})
+
+        if csv_import:
+            view.general.select_vm.select("Import a CSV file with a list of VMs to be migrated")
+            view.next_btn.click()
+            temp_file = tempfile.NamedTemporaryFile(suffix=".csv")
+            with open(temp_file.name, "w") as file:
+                headers = ["Name", "Provider"]
+                writer = csv.DictWriter(file, fieldnames=headers)
+                writer.writeheader()
+                for vm in vm_list:
+                    writer.writerow({"Name": vm.name, "Provider": vm.provider.name})
+            view.vms.hidden_field.fill(temp_file.name)
+        else:
+            view.next_btn.click()
+        view.vms.wait_displayed()
+
+        for vm in vm_list:
+            view.vms.filter("VM Name", vm.name)
+            for row in view.vms.table.rows():
+                if vm.name in row.vm_name.read():
+                    row[0].fill(True)
+            view.vms.clear_filters.click()
+        view.next_btn.click()
+
+        if pre_playbook:
+            view.advanced.pre_playbook.wait_displayed("5s")
+            view.advanced.pre_playbook.fill(pre_playbook)
+            view.advanced.pre_checkbox.click()
+        if post_playbook:
+            view.advanced.post_playbook.wait_displayed("5s")
+            view.advanced.post_playbook.fill(post_playbook)
+            view.advanced.post_checkbox.click()
+        view.next_btn.click()
+
+        if start_migration:
+            view.options.run_migration.select("Start migration immediately")
+        view.options.create.click()
+        wait_for(
+            lambda: view.results.msg.is_displayed, timeout=60, message="Wait for Results view"
+        )
+
+        base_flash = "Migration Plan: '{}'".format(name)
+        if start_migration:
+            base_flash = "{} is in progress".format(base_flash)
+        else:
+            base_flash = "{} has been saved".format(base_flash)
+        assert view.results.msg.text == base_flash
+        view.results.close.click()
+        return self.instantiate(name)
+
+
+@navigator.register(MigrationPlanCollection, "All")
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute("appliance.server", "LoggedIn")
+    VIEW = MigrationPlanView
+
+    def step(self):
+        if self.obj.appliance.version < "5.10":
+            self.prerequisite_view.navigation.select("Compute", "Migration")
+        else:
+            self.prerequisite_view.navigation.select("Compute", "Migration", "Migration Plans")
+
+
+@navigator.register(MigrationPlanCollection, "Add")
+class AddMigrationPlan(CFMENavigateStep):
+    VIEW = AddMigrationPlanView
+    prerequisite = NavigateToSibling("All")
+
+    def step(self):
+        self.prerequisite_view.wait_displayed()
+        self.prerequisite_view.create_migration_plan.click()
+
+
+@navigator.register(MigrationPlan, "Not Started")
```

Comment:
Required:
Please destination short name should be camel case and single word... `NotStarted` is better option.